GATEWAY/src/providers/ble.service.ts

```diff
@@ -22,11 +22,11 @@ export class BleService {
   };
 
   constructor(private events: Events,
-              private ble: BLE,
-              private devicesService: DevicesService,
-              private mqttClient: MqttClient,
-              private tilesApi: TilesApi,
-              private utils: UtilsService) {}
+              public ble: BLE,
```

Comment:
Testing purposes
Need to be able to get them one way or another
Can implement a get-method for these variables instead if that is better